docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def merlin(T, min_L=3, max_L=3, max_k=1, min_dist=None, decay_init_value=None, decay=None):
</pre><p>Rename this function as <code>murlin</code> ﻿</p><p> </p><p>Also, it feels like there are way too many input parameters here and <code>decay_init_value</code>  is a bad name. We want the names to be single words/letters where possible</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
> Rename this function as `murlin`

Will revise it. 

> Also, it feels like there are way too many input parameters here

We can remove `min_dist` as this is usually not known in advance. Also, I thought about using a tuple for `min_L` and `max_L`, something like `L_range = (min_L, max_L`).  We can allow it to be a single integer number if the user decides to get discords for a particular length only. 

(Btw, I think I should better change `L` to `m` to make it consistent with STUMPY).

What do you think?
 

> `decay_init_value` is a bad name.

I wanted the name to show its connection to `decay`. I will think about the name. I also think about the code itself and see if I can find a way to make it simpler.

Reply:
I'll be honest. I haven't actually looked at this final function in detail as I was too distracted by the other private functions. I want to delay looking at the public function until the private functions look solid.

Reply:
Totally makes senses...I will make a note about this comment in the notebook. So, I am going to resolve this now but I will keep the notes in the notebook so we do not forget.